helper/chksum.c

```diff
@@ -130,7 +130,7 @@ static inline int odph_process_l4_hdr(odp_packet_t      odp_pkt,
 		 * should come from the udp header, unlike for TCP where is
 		 * derived. */
 		l4_len            = odp_be_to_cpu_16(udp_hdr_ptr->length);
-		pkt_chksum_ptr    = (uint16_t *)(void *)&udp_hdr_ptr->chksum;
+		pkt_chksum_ptr    = (odp_una_u16_t *)(void *)&udp_hdr_ptr->chksum;
```

Comment:
Is the intermediate cast through (void *) needed for some reason? Cannot we just drop it now?

Reply:
fixed in v2